src/pandapipes/component_models/heat_consumer_component.py

```diff
@@ -71,6 +77,12 @@ def create_pit_branch_entries(cls, net, branch_pit):
         hc_pit[~np.isnan(mdot), MDOTINIT] = mdot[~np.isnan(mdot)]
         treturn = net[cls.table_name()].treturn_k.values
         hc_pit[~np.isnan(treturn), TOUTINIT] = treturn[~np.isnan(treturn)]
+        mask_q0 = hc_pit[:, QEXT] == 0 & np.isnan(hc_pit[:, MDOTINIT])
+        hc_pit[mask_q0, ACTIVE] = False
```

Comment:
small adaption here:
- use np.any
- put this line after the if-statement

Reply:
Done

Reply:
Done!